transformers/browserchannel/server.js

```diff
@@ -31,6 +31,7 @@ module.exports = function server() {
       , socket.query                            // Optional query string.
       , socket.id                               // Unique connection id.
     );
+    spark.socket = socket;
```

Comment:
Can we make the spark constructor accept the a socket argument instead of adding it to the instance? as suggested by @3rd-Eden?